src/KygekTeam/KygekPingTPS/Main.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * See your and others's ping and the server tps
+ * Copyright © 2021 KygekTeam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ */
+declare(strict_types=1);
+namespace KygekTeam\KygekPingTPS;
+
+use pocketmine\plugin\PluginBase;
+use pocketmine\utils\TextFormat;
+
+use JackMD\UpdateNotifier\UpdateNotifier;
+
+class Main extends PluginBase {
+	private static $instance;
+
+	public static function getInstance() : self {
+		return self::$instance;
+	}
+
+	public function onEnable() {
+		self::$instance = $this;
+		$this->saveDefaultConfig();
+		$this->checkConfig();
+		
+        if ($this->getConfig()->get("check-updates", true)) {
+        	if (class_exists("\JackMD\UpdateNotifier\UpdateNotifier")) {
+            	\JackMD\UpdateNotifier\UpdateNotifier::checkUpdate($this->getDescription()->getName(), $this->getDescription()->getVersion());
+            } else {
+            	$this->getLogger()->notice("JackMD\UpdateNotifier is not installed on your server, install it or turn off check-updates on your config");
```

Comment:
This is unnecessary and will likely break/violate the Poggit rules. UpdateNotifier is bundled with the plugin using virion system anyway.